test/sema/dm_option.f90

```diff
@@ -0,0 +1,15 @@
+!
+! Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+! See https://llvm.org/LICENSE.txt for license information.
+! SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+! Just check some of the macros - if they are printed
+! RUN: %flang -dM -E %s | FileCheck %s --check-prefix=CHECK-MACROS
```

Comment:
Add flags to ensure target system and architecture here.

Reply:
flags added